vm/class.go

```diff
@@ -1076,6 +1098,19 @@ func (c *RClass) setAttrAccessor(args interface{}) {
 	c.setAttrWriter(args)
 }
 
+func (c *RClass) ancestors() []*RClass {
```

Comment:
@hanachin Hice!

Reply:
Looks like you can also implement the `ancestors` method later very easily?

Reply:
Yes! 🍻